### ISSUE TITLE
Trend chart refinement

### ISSUE THREAD

**Issue by amberwreed** [18F/crime-data-api#308]
- [x] Make adjustments based on user testing (for example, add time period to chart?)
- [x] Create production ready sketch files
- [x] Walk-through designs with FE team
- [ ] FE design implement trend chart per annotated sketch file
- [ ] Visual design review for design consistency

**Comment by jeremiak** [comment]
I believe this is ready to be reviewed

**Comment by LarryBafundo** [comment]
@jeremiak are we waiting on revised deigns in order to close this issue? i'm referring to the last two (unchecked) boxes in the to do list.

**Comment by amberwreed** [comment]
Thanks @jeremiak! Looks good. Here are a couple of outstanding items for the trend chart. I've tried to prioritize. Pls look over the lower priority items because some of them maybe easy fixes but will not make or break the experience.

High priority:
- [ ] Missing source of chart
- [ ] Trend chart should use 4 horizontal bars and the trend line shouldn't ever go above the top horizontal bar.
<img width="800" alt="screen shot 2017-02-06 at 9 15 42 pm" src="https://cloud.githubusercontent.com/assets/13576520/22678802/b359e914-ecb1-11e6-948e-66fe5369baa6.png">
- [ ] Move the download data to below the chart. See latest annotated file.


Lower priority:
- [ ] Use the the download icon provide in the icon file on drive
- [ ] Title color is looking wrong to me. We spec'd #DFE6ED. Can we try #DCE6F0?
- [ ] Title height is ~145px, should be ~128px
- [ ] Selected year should be bolded. For example, if you mouse over 2008, 2008 should be bold at the bottom of the selected line.
- [ ] Missing Trend line name at the start of the trend line. See annotated file.
- [ ] Increase the left margin of the trend chart to make room for the trend chart names and to decrease the space between the first year and the 0 on the y axis.
- [ ] Missing button to zoom in on area of interest. See annotated file.

**Comment by jeremiak** [comment]
Thank you @amberwreed, that is super helpful. I think the order looks good so we'll work through those.

**Comment by amberwreed** [comment]
Here's a link to the latest file. I moved the download link out of the header (to match how we are handling the downloads of the NIBRS charts). https://drive.google.com/open?id=0B5cw7Ca2CjsySUhicVVMSWxBSDA

**Comment by amberwreed** [comment]
@jeremiak @brendansudol Making sure you see this. ^^

**Comment by jeremiak** [comment]
High priority UI bugs have been recorded as individual issues:
* [#255 - Trend chart line shouldn't ever go above the top horizontal bar](https://github.com/18F/crime-data-explorer/issues/255)
* [#254 - Missing source of trend chart data](https://github.com/18F/crime-data-explorer/issues/254)

Given that, should this card be closed? Perhaps the lower priority bugs can be stored in yet another new issue and put in the backlog for UI fixes we want to do, time permitting?
